deploy.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+VAULT_TOKEN=$1
+GIT_BRANCH=$2
+
+#need to get the environment from the branch name
+if [ "$GIT_BRANCH" == "develop" ]; then
+	ENVIRONMENT="dev"
+elif [ "$GIT_BRANCH" == "alpha" ]; then
+        ENVIRONMENT="alpha"
+elif [ "$GIT_BRANCH" == "staging" ]; then
+	ENVIRONMENT="staging"
+elif [ "$GIT_BRANCH" == "master" ]; then
+	ENVIRONMENT="prod"
+elif [ "$GIT_BRANCH" == "gawb3639" ]; then
+	ENVIRONMENT="dev"
+else
+	echo "Unknown Git branch $GIT_BRANCH"
+	#exit 1
+fi
+
+GOOGLE_PROJECT=broad-avram-$ENVIRONMENT
+SERVICE_VERSION=version1
+
+# pull the credentials for the service account
+docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN broadinstitute/dsde-toolbox vault read --format=json "secret/dsde/avram/$ENVIRONMENT/deploy-account.json" | jq .data > deploy_account.json
+
+# Auth with deploy service account
+gcloud auth activate-service-account --key-file=deploy_account.json
```

Comment:
Is it okay for us to be running `gcloud` commands like this or should we be doing it inside a docker container like bond does? https://github.com/DataBiosphere/bond/blob/develop/deploy.sh#L33
Attn: @jacmrob

Reply:
I checked with Cloud Accounts and Matt Bemis said that using docker is preferable to avoid interference between jobs. I am working on the conversion.